README.md

```diff
@@ -7,3 +7,29 @@ Allow Journal Managers and (optionally) Authors to deposit articles via the SWOR
 This plugin is best installed using the Plugin Gallery from within Open Journal Systems.
 Log in as Administrator, navigate to `Settings` > `Website` > `Plugins`, and open the "Plugin Gallery"
 tab. Find the Sword plugin listed there and use the `Install` button to install it.
+
+## Configuration
+
+### General Settings
+- After successfull installation, you  have to **enable** the plugin in both generic and inmport export plgun section.
+- A new tab, **Sword Settings** will be available next to Website Settigns -> plugins
+- After clicking the tab, ad a new Peposit point using the **Create Deposit Point**
+- Add your Service Provider Information there. 
+  -   Name: Customized name
+  -   Deposit Point URL: Service Document name :   e.g.http://demo.dspace.org/swordv2/servicedocument
+  -   Username: Username
+  -   Password: Password
+  -   API Key: Some repository providers may supply you an API key.
+  -   Type: select your requirement.
+-   Save and Exit
+  
+ ### Import/Export Plugin
+ - Select Sword Plugin in Import/Export Plugins
+ - Got to **Import/Export Data**
+ - Select the **configured**  Deposit Point
+ - Avaliable Collections will be available under **Deposit Point**
+ - Choose the oprions either  Deposit Galleys / Deposit Most Recent Editorial Fil or both.
+ - Select the artciles
```

Comment:
*articles

Reply:
@asmecher ; Thanks a lot. Sorry for the trouble , I have corrected all the issues and resolved the others myself.